modules/op-release-notes-1-8.adoc

```diff
@@ -50,8 +201,55 @@ Workaround: Use `nodejs:14-ubi8` instead of the `nodejs:14-ubi8-minimal` image s
+// link:https://github.com/tektoncd/pipeline/pull/4651
+// Lee Bernick @lbernick
+
+* This update changes how resource requirements are calculated when one or more `LimitRange` objects are present in the namespace where a `TaskRun` object is. With this update, the scheduler considers `step` containers and excludes all other app containers, such as sidecar containers, when factoring requests from `LimitRange` objects.
+// link:https://github.com/tektoncd/pipeline/issues/4996
+// Simon Kaegi @skaegi
+
+* Before this update, under specific conditions, the flag package might incorrectly parse a subcommand immediately following a double dash flag terminator, `--`. In that case, it ran the entrypoint subcommand instead of the actual command. This update fixes this flag-parsing issue so that the entrypoint runs the correct command.
+// link:https://github.com/tektoncd/pipeline/issues/5095
+// Vincent Demeester @vdeemester
+
+* Before this update, the controller might generate multiple panics if pulling an image failed, or its pull status was incomplete. This update fixes the issue by checking the `step.ImageID` value  instead of the `status.TaskSpec` value.
+// link:https://github.com/tektoncd/pipeline/pull/5245(#5245)
+// Piyush Garg @piyush-garg
+
+* Before this update, canceling a pipeline run that contained an unscheduled custom task produced a `PipelineRunCouldntCancel` error. This update fixes the issue. You can cancel a pipeline run that contains an unscheduled custom task without producing that error.
+// link:https://github.com/tektoncd/pipeline/issues/5295(#5295)
+//Andrew Bayer @abayer
+
+
+* Before this update, if the `<NAME>` in `$params["<NAME>"]` or `$params['<NAME>']` contained a dot character (`.`),  any part of the name to the right of the dot was not extracted. For example, from `$params["org.ipsum.lorem"]`, only `org` was extracted.
++
+This update fixes the issue so that `$params` fetches the complete value. It also throws an error if `<NAME>` is not enclosed in single or double quotes. For example:
++
+* `$params.org.ipsum.lorem` is not valid and generates a validation error.
+* `$params["org.ipsum.lorem"]` and `$params['org.ipsum.lorem']` are valid and the entire value of `<NAME>`, `org.ipsum.lorem`, is extracted.
+// https://github.com/tektoncd/pipeline/pull/4880
+
+// .Triggers
 
-// Pipelines
-// *
+With this update, `Trigger` resources support custom interceptors and ensure that the port of the custom interceptor service is the same as the port in the `ClusterInterceptor` definition file.
+// link:https://github.com/tektoncd/triggers/pull/1383[(#1383)]
+// @savitaashture
```

Comment:
@savitaashture Please review and comment. Can we add a brief description of the problem before the fix? Thanks.

Reply:
Please ignore the fact that GitHub closed this PR and leave your review here anyway.